File: Examples/Mvc.VB/Controllers/DocumentViewerController.UsingLayout.vb
Imports GleamTech.DocumentUltimate.AspNet.UI

Namespace Controllers
    Partial Public Class  DocumentViewerController
        Inherits Controller

        Public Function UsingLayout() As ActionResult
            Dim documentViewer = New DocumentViewer() With {
                    .Width = 800,
                    .Height = 600,
                    .Resizable = True,
                    .Document = "~/App_Data/ExampleFiles/Default.pdf"
                    }

            Return View(documentViewer)
        End Function

    End Class
End Namespace

File: Examples/Mvc.CS/Controllers/DocumentViewerController.UsingLayout.cs
using System.Web.Mvc;
using GleamTech.DocumentUltimate.AspNet.UI;

namespace GleamTech.DocumentUltimateExamples.Mvc.CS.Controllers
{
    public partial class DocumentViewerController
    {
        public ActionResult UsingLayout()
        {
            var documentViewer = new DocumentViewer
            {
                Width = 800,
                Height = 600,
                Resizable = true,
                Document = "~/App_Data/ExampleFiles/Default.pdf"
            };

            return View(documentViewer);
        }
    }
}